src/xdocs/config_javadoc.xml

```diff
@@ -4444,6 +4445,9 @@ public class Test {
 public class Test { // violation as required tag is missed
   /** some doc */
   void foo() {} // OK, as methods are not checked by default
+}
+
+class Test2 { // OK, no javadoc
```

Comment:
All java examples should be same. Make it single line and copy in all other examples